### ISSUE TITLE
Package build: no visible binding for global variable

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/TLF-Library#324]
```
* checking R code for possible problems ... NOTE
.mergeLegend: no visible binding for global variable 'color'
.mergeLegend: no visible binding for global variable 'shape'
.mergeLegend: no visible binding for global variable 'size'
.mergeLegend: no visible binding for global variable 'linetype'
.mergeLegend: no visible binding for global variable 'fill'
plotTimeProfile: no visible binding for global variable 'fillLength'
plotTimeProfile: no visible binding for global variable
  'linetypeLength'
setCaptionColor: no visible binding for global variable 'newPlotObject'
setCaptionFill: no visible binding for global variable 'newPlotObject'
setCaptionLabels: no visible binding for global variable
  'newPlotObject'
setCaptionLinetype: no visible binding for global variable
  'newPlotObject'
setCaptionOrder: no visible binding for global variable 'newPlotObject'
setCaptionShape: no visible binding for global variable 'newPlotObject'
setCaptionSize: no visible binding for global variable 'newPlotObject'
setCaptionVisibility: no visible binding for global variable
  'newPlotObject'
Undefined global functions or variables:
  color fill fillLength linetype linetypeLength newPlotObject shape
  size
```

**Comment by IndrajeetPatil** [comment]
Related to https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/162

**Comment by Yuri05** [comment]
>Related to https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/162

actually included in #162. closing
